_includes/components/tag.html

```diff
@@ -0,0 +1,7 @@
+<div class="tag-container">
+  <div class="tag {{ include.additionalClasses }}">
```

Comment:
Think it's be worth adding a data attribute to this for some kinda filtering redundancy?
```suggestion
  <div class="tag {{ include.additionalClasses }}" data-tag="{{ include.text }}">
```